lang/en.json

```diff
@@ -788,8 +788,12 @@
     "description": "Error text telling user to update firmware"
   },
   "webusb-not-supported": {
-    "defaultMessage": "Unfortunately, WebUSB is not supported in this browser and your program will be saved instead. We recommend Google Chrome or Microsoft Edge so you can connect directly to your micro:bit.",
-    "description": "Shown when user's browser doesn't support WebUSB"
+    "defaultMessage": "Unfortunately, WebUSB is not supported in this browser and your program will be saved to your computer instead. Follow the steps on the next screen to transfer it. We recommend Google Chrome or Microsoft Edge so you can connect directly to your micro:bit.",
```

Comment:
Should it be "Follow the steps on the next screen to transfer it to your micro:bit."?

Reply:
Oops.